### DISCUSSION TITLE
Document-Level Permission Settings Request

### DISCUSSION THREAD

**Discussion by dariofflow** [outline/outline#6829] [Ideas]
We have identified a need for more nuanced permission settings within our document collections. Our goal is to enhance collaboration while maintaining the necessary levels of privacy and access control within our organization.

Feature Request:
We would like the ability to set permissions on individual documents and their subpages, similar to the existing permission options available for collections. This would enable us to assign specific access rights (view, edit, or none) to individual users or groups on a per-document basis.

Use Case:
In a corporate environment with a diverse team, it’s crucial to manage document access efficiently. Certain documents may contain sensitive information that only specific team members should edit or view. The ability to fine-tune access on a document level can prevent information over-share and ensure that team members only see what's relevant to their roles.

Current Limitation:
As of now, permissions can only be set at the collection level, which applies broadly to all documents within. This does not accommodate the need for differential access based on document content or team function.

We believe that implementing this feature would significantly benefit teams like ours that use Outline for knowledge management and would greatly appreciate your consideration of this enhancement.

Thank you for developing Outline and for your ongoing support of the community!

**Comment by tommoor** [comment]
This feature already exists. [See documentation here](https://docs.getoutline.com/s/guide/doc/sharing-LG2sGOLIpl).
